tools/empath-split.py

```diff
@@ -59,23 +74,35 @@ def parse_args():
 enabling/disabling options. Run 'wasm-split -h' for the list of options. But you
 should NOT add --manifest, because this will be generated from this script.
 """)
-  parser.add_argument('wasm', help='Path to the input wasm file')
-  parser.add_argument('paths_file', help='Path to the input file containing paths')
+  parser.add_argument('wasm', nargs='?', help='Path to the input wasm file')
+  parser.add_argument('paths_file', nargs='?', help='Path to the input file containing paths')
```

Comment:
These became optional because when you use `--print-sources` these are not mandatory. (We manually check for their existence below when `--print-sources` is not given)